src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerUpdateLanguage.java

```diff
@@ -28,8 +28,8 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         new PageBuilder(configBuilder
             .event(CASEWORKER_UPDATE_LANGUAGE)
             .forAllStates()
-            .name("Update language")
-            .description("Update language")
+            .name("Update Language")
```

Comment:
why?

Reply:
Corrected it, not sure why I got this during merging.